pytorch_lightning/accelerators/accelerator.py

```diff
@@ -388,3 +389,11 @@ def all_gather(self, tensor: Union[torch.Tensor], group: Optional[Any] = None, s
             A tensor of shape (world_size, batch, ...)
         """
         return all_gather_ddp_if_available(tensor, group=group, sync_grads=sync_grads)
+
+    def process_dataloader(self, dataloader: Union[Iterable, DataLoader]) -> Union[Iterable, DataLoader]:
```

Comment:
now it is duplicate to Plugin's `process_dataloader`?

Reply:
It is the intended behaviour. Ideally, the trainer should interact only with the accelerator, so `process_dataloader` from accelerator redirect to training_type_plugin.